src/index.ts

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import path from 'path';
+
+import { router } from './routers/user-router';
+const app = express();
+
+app.set('port', (process.env.PORT || 4200));
+
+app.use(express.static(path.join(__dirname, '../src')));
+app.use(express.json());
+
+app.use((req, res, next) => {
+    res.header('Access-Control-Allow-Origin', '*');
```

Comment:
What reason to allow CORS requests from any(*) resources?

Reply:
delete